rust/types/src/rfq.rs

```diff
@@ -57,54 +58,159 @@ pub enum RequestForQuoteUpdate {
+        #[serde(rename = "s")]
+        symbol: String,
+        #[serde(rename = "S")]
+        side: Side,
+        #[serde(rename = "q", skip_serializing_if = "Option::is_none")]
+        quantity: Option<Decimal>,
+        #[serde(rename = "Q", skip_serializing_if = "Option::is_none")]
+        quote_quantity: Option<Decimal>,
+        #[serde(rename = "w")]
+        submission_time: i64,
+        #[serde(rename = "W")]
+        expiry_time: i64,
+        #[serde(rename = "X")]
+        order_status: OrderStatus,
+        #[serde(rename = "T")]
+        timestamp: i64,
+    },
+    #[serde(rename = "rfqAccepted")]
+    RfqAccepted {
+        #[serde(rename = "E")]
+        event_time: i64,
+        #[serde(rename = "R")]
+        rfq_id: u64,
+        #[serde(rename = "C", skip_serializing_if = "Option::is_none")]
+        client_id: Option<u32>,
+        #[serde(rename = "s")]
+        symbol: String,
+        #[serde(rename = "S")]
+        side: Side,
+        #[serde(rename = "q", skip_serializing_if = "Option::is_none")]
+        quantity: Option<Decimal>,
+        #[serde(rename = "Q", skip_serializing_if = "Option::is_none")]
+        quote_quantity: Option<Decimal>,
+        #[serde(rename = "w")]
+        submission_time: i64,
+        #[serde(rename = "W")]
+        expiry_time: i64,
+        #[serde(rename = "X")]
+        order_status: OrderStatus,
+        #[serde(rename = "T")]
+        timestamp: i64,
+    },
+    #[serde(rename = "rfqCancelled")]
+    RfqCancelled {
+        #[serde(rename = "E")]
+        event_time: i64,
+        #[serde(rename = "R")]
+        rfq_id: u64,
+        #[serde(rename = "C", skip_serializing_if = "Option::is_none")]
+        client_id: Option<u32>,
+        #[serde(rename = "s")]
+        symbol: String,
+        #[serde(rename = "S")]
+        side: Side,
+        #[serde(rename = "q", skip_serializing_if = "Option::is_none")]
+        quantity: Option<Decimal>,
+        #[serde(rename = "Q", skip_serializing_if = "Option::is_none")]
+        quote_quantity: Option<Decimal>,
+        #[serde(rename = "w")]
+        submission_time: i64,
         #[serde(rename = "W")]
         expiry_time: i64,
         #[serde(rename = "X")]
         order_status: OrderStatus,
         #[serde(rename = "T")]
         timestamp: i64,
     },
+    #[serde(rename = "quoteAccepted")]
     QuoteAccepted {
         #[serde(rename = "E")]
         event_time: i64,
         #[serde(rename = "R")]
         rfq_id: u64,
         #[serde(rename = "u")]
         quote_id: u64,
+        #[serde(rename = "C", skip_serializing_if = "Option::is_none")]
+        client_id: Option<u32>,
+        #[serde(rename = "s")]
+        symbol: String,
+        #[serde(rename = "p", skip_serializing_if = "Option::is_none")]
+        price: Option<Decimal>,
         #[serde(rename = "X")]
         order_status: OrderStatus,
         #[serde(rename = "T")]
         timestamp: i64,
     },
+    #[serde(rename = "quoteCancelled")]
```

Comment:
Hm surprised everythinng is camelcase.

But in that case should we simply do at the top struct: `#[serde(rename_all = "camelCase")]`